rpc/ethereum/backend/feebackend.go

```diff
@@ -61,14 +61,15 @@ func (e *EVMBackend) processBlock(
 	rewardCount := len(rewardPercentiles)
 	targetOneFeeHistory.Reward = make([]*big.Int, rewardCount)
 	for i := 0; i < rewardCount; i++ {
-		targetOneFeeHistory.Reward[i] = big.NewInt(2000)
```

Comment:
`2000` was chosen as default in https://github.com/tharsis/ethermint/pull/734/files#diff-4d996264fd51e2936260254650a034814a870ddfa9f7df7398d53feba7c39ef6R64.
Standard default value is `0` and it was usually rewritten anyway by the code below this line.